### PR TITLE
Add descriptions for confusion matrix metrics and TNR

### PR DESCRIPTION
**Pull request by Leselesel1234**
Description of confusion matrix values
Definition table TP, FP, TN, FN
Description Specificity/TNR 
Illustrating Example table
Sample output with default values 0.95, 1 (subject to change) cm, pos1 , pos2, default

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,6 +86,42 @@ to screen to detect this record (TD); and (2) the Average Time to Discover
...
+
+|                      | Definition                                                                             | Calculation                     |   
+|----------------------|----------------------------------------------------------------------------------------|---------------------------------|
+| True Positives (TP)  | The number of relevant records found at recall level.                                  | Relevant Records * r%           |      
+| False Positives (FP) | The number of irrelevant records reviewed at recall level.                             | Records Reviewed – TP           |
+| True Negatives (TN)  | The number of irrelevant records correctly not reviewed at recall level.               | Irrelevant Records – FP         |  
+| False Negatives (FN) | The number of relevant records not reviewed at recall level (missing relevant records) | Relevant Records – TP           | 
+
+
+Specificity / True Negative Rate (TNR)
```
Comment (Leselesel1234) [inline README.md:104]: Remove title (?)

File: README.md [hunk 2]
```diff
@@ -459,6 +565,7 @@ is the results of the metric. Some metrics are computed for multiple values.
 | `erf` | Labels | ERF | 0.10 |
 | `atd` | Average time to discovery (in label actions) | - | - |
 | `td` | Row number (starting at 0) | Number of records labeled | - |
+| `cm` | Recall  | Confusion matrix values at recall | 0.95, 1 |
```
Comment (Leselesel1234) [inline README.md:569]: I now show the 0.95 and 1 as default intercepts in sample output, however this does not align with current code
Another small issue is that with the command --cm 0.95 1, TNR is outputted at those intercepts as well.

File: README.md [hunk 3]
```diff
@@ -442,10 +478,80 @@ which results in
...
+                    [
+                        1.0,
+                        0.915867
+                    ]
+                ]
             }
         ]
     }
-}
+}
```
Comment (J535D165) [inline README.md:554]:
```suggestion
}
```

File: README.md (hunk 2 above)
Comment (J535D165) [reply README.md:569]: Oke, I see your issue regarding readability. This should be resolved later with some API changes.

**Review by J535D165 — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -86,6 +86,42 @@ to screen to detect this record (TD); and (2) the Average Time to Discover
...
+|----------------------|----------------------------------------------------------------------------------------|---------------------------------|
+| True Positives (TP)  | The number of relevant records found at recall level.                                  | Relevant Records * r%           |      
+| False Positives (FP) | The number of irrelevant records reviewed at recall level.                             | Records Reviewed – TP           |
+| True Negatives (TN)  | The number of irrelevant records correctly not reviewed at recall level.               | Irrelevant Records – FP         |  
+| False Negatives (FN) | The number of relevant records not reviewed at recall level (missing relevant records) | Relevant Records – TP           | 
+
+
+Specificity / True Negative Rate (TNR)
+
+[Kusa et al. (2023)](https://www.sciencedirect.com/science/article/pii/S2667305323000182) propose to normalize the commonly used WSS measure for class imbalance (nWSS).
```
Comment (J535D165) [inline README.md:106]:
```suggestion
Specificity / True Negative Rate (TNR). [Kusa et al. (2023)](https://www.sciencedirect.com/science/article/pii/S2667305323000182) propose to normalize the commonly used WSS measure for class imbalance (nWSS).
```

**Pull request merged**